docs/mcp/tweakcn.md

```
@@ -0,0 +1,50 @@
+
+The `aidesigner` CLI now prompts for both when you opt into the server during project scaffolding so the values land in `.mcp.json` automatically.【F:bin/aidesigner†L793-L852】【F:bin/aidesigner†L1528-L1573】 You can also define them in your shell or `.env` file before rerunning `npx aidesigner init`.
+
+## 3. Smoke test the orchestrator
+
+Once the entry exists, launch the orchestrator and ask it to list the configured MCP servers. TweakCN should appear alongside the other integrations:
+
+```bash
+node .dev/tools/cli.js mcp list
+```
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)